src/RestaurantsContainer.jsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Link } from 'react-router-dom';
+
+import { loadRestaurant, setRestaurant } from './actions';
 
 import { get } from './utils';
 
 export default function RestaurantsContainer() {
+  const dispatch = useDispatch();
+
   const restaurants = useSelector(get('restaurants'));
 
+  function handleClick(restaurantId) {
+    dispatch(setRestaurant({ id: restaurantId }));
```

Comment:
id 라고만 받아서 넘겨주는 건 어떤가요?
```suggestion
  function handleClick(id) {
    dispatch(setRestaurant({ id }));
```

Reply:
id로 코드를 간략하게 줄일 수 있겠지만, 
이게 어떤 의미를 가진 id인지 표시하고 싶어서 번거롭지만, restaurantId로 작성했습니다. 

추 후에 handleClick을 다시 들여다 볼 때, 생각을 한 번 더 하지 않아도 되는... 

readability > brevity 라는 기준을 잡고 진행해봤습니다. :) 

줏대가 아직 갈대 같아서, 저만의 기준점을 한번 정리해봐야겠네요 ㅎ

Reply:
본인의 의도를 드러내는 이유라면 좋은 것 같습니다 👍🏼 👍🏼